src/components/memo-panel.tsx

```diff
@@ -57,7 +57,7 @@ export const MemoPanel: React.FunctionComponent = () => {
   const repo: any = useRepository()
   const classes: any = useStyles()
   const [expanded, setExpanded] = useState<string | false>(false)
-  const [editmode, setEditmode] = useState<string | false>('')
+  const [editmode, setEditmode] = useState<string>('')
   const [editText, setEditText] = useState<string>('')
```

Comment:
Please use useState like this `const [editmode, setEditmode] = useState('')` when initial value is the type of the state value.